docs/index.html

```diff
@@ -792,6 +794,26 @@ <h3 class="i18n" id="trx_color">Custom Colors</h3>
     </p>
     <hr>
 
+    <h2 class="i18n">Finding Transactions</h2>
+        <div class="row">
+        </div>
+            <p class="i18n" id="trx_search">
+                The MMEX desktop application provides several utilities to help users find specific transactions. The most powerful of these tools is the Transaction Reports
+                feature described in more detail <a class="i18n" href="#section21.1">here</a>. In addition, there are several other utilities which
+                can be used to find transactions.
+            </p>
+        
+        <hr>
+    
+        <h3 class="i18n" id="trx_search-1">Searching from the Account View</h3>
+            <div class="row">
+            </div>
+                <p class="i18n">
+                    Placeholder text.
```

Comment:
In general the PR is OK.
Please provide real text instead "Placeholder text." the commit it.
As result this changes apeears here.

Reply:
Most recent commit (ab09cd5) should have content with correctly styled headers. I titled the section "Finding Information" and included both finding transactions and finding payees under the heading, with the general goal that this could be a general search-related topic in the documentation. If that is not desired, please let me know and I can reformat.